src/components/Status/tests/__snapshots__/Status.test.js.snap

```diff
@@ -10,3 +10,38 @@ exports[`<Status /> renders correctly 1`] = `
   off status
 </Fragment>
 `;
+
+exports[`<StatusField> renders correctly 1`] = `
+<Fragment>
+  <Icon
+    className="kubevirt-status__icon"
+    name="off"
+    type="pf"
+  />
+  <div>
+    some content
+  </div>
+</Fragment>
+`;
+
+exports[`<StatusField> renders correctly 2`] = `
+<Fragment>
+  <div>
+    some important content
+  </div>
+</Fragment>
+`;
+
+exports[`<StatusField> renders correctly 3`] = `
+<Fragment>
+  <div>
+    some less important content
+  </div>
+</Fragment>
+`;
+
+exports[`<StatusField> renders correctly 4`] = `<Fragment />`;
```

Comment:
renders `Fragment` only?

Reply:
@suomiy , Thanks for the  reviews :+1: , removed the unnecessary checks :beers: :cake: